docs/index.rst

```diff
@@ -31,6 +31,6 @@ Snapcraft is a member of the Canonical family. It's an open source project
 that warmly welcomes community projects, contributions, suggestions, fixes
 and constructive feedback.
 
-* `Ubuntu Code of Conduct <https://ubuntu.com/community/code-of-conduct>`_
+* `Ubuntu Code of Conduct <https://ubuntu.com/community/code-of-conduct>`_.
 * `Canonical contributor licenses agreement
-  <https://ubuntu.com/legal/contributors>`_
+  <https://ubuntu.com/legal/contributors>`_.
```

Comment:
Not sure why these were added, but I'll revisit when I review the home page.

Reply:
It does look odd - it seems like they've always been in starbase.